starknet_py/utils/typed_data.py

```diff
@@ -18,6 +20,55 @@ class Parameter:
     name: str
     type: str
 
+    def to_dict(self) -> dict:
```

Comment:
Do we need this method for the `Parameter` class?

Reply:
No, it's not used anywhere, removed